designsafe/static/scripts/projects/components/edit-project/edit-project.component.js

```diff
@@ -425,23 +431,28 @@ class EditProjectCtrl {
             projectData.keywords = this.form.keywords;
         }
 
-        this.savePrj(projectData).then((project) => {
-            if (this.project) {
-                this.project.value = project.value;
-            }
-            if (!this.form.uuid) {
-                this.$state.go(
-                    'projects.view.data',	
-                    {	
-                        projectId: project.uuid,	
-                        filePath: '/',	
-                        projectTitle: project.value.title	
-                    },	
-                    {reload: true}	
-                );
+        this.modalInstance.result.then((res) => {
```

Comment:
This modalInstance response handler should be in the same if block as where it is instantiated, since if it is never instantiated this throws an error, i.e. when the user does not remove themselves.

The `this.savePrj` can stay down here as it was before.

Reply:
Nah I’ll get it sorted out tomorrow morning. We can go to dev without it.